tutorial/markdown/java/spring-boot.md

```diff
@@ -53,172 +91,494 @@ mvn package
+
+The connection string should be in the following format:
 
-### Dependency Injection via DBSetupRunner Class
+```properties
+spring.couchbase.bootstrap-hosts=couchbases://cb.fwu-oviwcpjq4v.cloud.couchbase.com
+OR
+spring.couchbase.bootstrap-hosts=localhost
+```
+
+The couchbases protocol is used for secure connections.
 
-The quickstart code provides a CommandLineRunner called DBSetupRunner in the runners folder that wires up the Bucket and Cluster objects for dependency injection. This runner also creates the bucket, collection, scope, and indexes for the tutorial to run properly automatically when the application starts.
+```properties
+spring.couchbase.bootstrap-hosts=DB_CONN_STR
+spring.couchbase.bucket.user=DB_USERNAME
+spring.couchbase.bucket.password=DB_PASSWORD
+```
 
 ## Running The Application
 
+### Directly on the machine
+
 At this point the application is ready, and you can run it via your IDE or from the terminal:
 
 ```shell
 mvn spring-boot:run -e -X
 ```
 
-> Note: Couchbase Server 7 must be installed and running on localhost (http://127.0.0.1:8091) prior to running the Spring Boot app.
+<!-- Screeenshot -->
+
+![Spring Boot Application](./app-startup-spring-boot.png)
+
+> Note: Either the Couchbase Server must be installed and running on localhost or the connection string must be updated in the `application.properties` file.
 
 Once the site is up and running you can launch your browser and go to the Swagger Start Page]: `http://localhost:8080/swagger-ui/` to test the APIs.
 
+<!-- Screenshot -->
+
+![Swagger UI](./swagger-documentation-spring-boot.png)
+
+### Docker
+
+You can also run the application using Docker. The `Dockerfile` is provided in the root directory of the project. To build and run the Docker image, use the following commands:
+
+```shell
+docker build -t springboot-couchbase .
+```
+
+```shell
+docker run -p 8080:8080 springboot-couchbase
+```
+
 ## What We'll Cover
 
-A simple REST API using Spring Boot and the Couchbase SDK version 3.x with the following endpoints:
+A simple REST API using Spring Boot and the `Couchbase SDK version 3.x` with the following endpoints:
 
-- [POST a Profile](#post-a-profile) – Create a new user profile
-- [GET a Profile by Key](#get-a-profile-by-key) – Get a specific profile
-- [PUT Profile](#put-profile) – Update a profile
-- [DELETE Profile](#delete-profile) – Delete a profile
-- [GET Profiles by Searching](#get-profiles-by-searching)  – Get all profiles matching first or last Name
+- [Create new airlines with essential information](#post-mapping).
+- [Update airline details](#put-mapping).
+- [Delete airlines](#delete-mapping).
+- [Retrieve airlines by ID](#get-mapping).
+- [List all airlines with pagination](#get-mapping).
+- [List airlines by country](#custom-sql-queries).
+- [List airlines by destination airport](#custom-sql-queries).
```

Comment:
Period is not required at the end. 
Also, why does it say mapping instead of airline. And you have linked to the same section for two end points.

Reply:
removed periods at the end.
Mapping because they are mapping HTTP GET,PUT,PATCH,DELETE requests onto specific handler methods.